editor/src/components/canvas/canvas-types.ts

```diff
@@ -649,13 +669,22 @@ type ZoomUI = {
   zoomIn: boolean
 }
 
+type SetUsersPreferredStrategy = {
+  action: 'SET_USERS_PREFERRED_STRATEGY'
+  strategyName: string // TODO limit it to string literal union of registered strategy names?
```

Comment:
I think this TODO should be done. In a way, that a strategy has a type, which is a string literal union, and a user digestible name, which can be shown in a ui, but not used as an identifier.

Reply:
so you propose to add a new field to the strategy, which is the non-user-facing strategy name? and restrict this type here to only valid string literals, right?

Reply:
update: we decided to only fix this after this is merged to master

Reply:
Note: we postponed this because there are no actual strategies in this PR